graphics/src/ColladaExporter.cc

```diff
@@ -165,14 +183,6 @@ void ColladaExporter::Export(const Mesh *_mesh, const std::string &_filename,
   this->dataPtr->path = unix_filename.substr(0, beginFilename);
   this->dataPtr->filename = unix_filename.substr(beginFilename);
 
-  if (this->dataPtr->materialCount != 0 &&
-      this->dataPtr->materialCount != this->dataPtr->subMeshCount)
-  {
-    ignwarn << "Material count [" << this->dataPtr->materialCount <<
-        "] different from submesh count [" <<
-        this->dataPtr->subMeshCount << "]\n";
-  }
-
```

Comment:
Is this check performed somewhere else now?  I can't tell why it was dropped from here.

Reply:
This check is no longer required. There is no longer a one to one relationship between submeshes and materials. Submeshes now can reuse materials used by other submeshes.